src/desktop/components/credit_card/client/error_handling_form.coffee

```diff
@@ -38,7 +38,7 @@ module.exports = class ErrorHandlingForm extends Backbone.View
       continue unless val.el && !val.validator(val.el)
       errors[key] = val.message || "Invalid #{val.label || key}"
```

Comment:
It seems that the `Invalid #{val.label}` section of this line is no longer relevant. We could replace with `'This field is required'` if we end up needing this and the other error message. Otherwise we could get rid of that string here.